x-pack/plugins/security_solution/common/search_strategy/security_solution/hosts/index.ts

```diff
@@ -16,9 +16,11 @@ export * from './uncommon_processes';
 
 export enum HostsQueries {
   authentications = 'authentications',
+  authenticationsSummary = 'authenticationsSummary',
```

Comment:
Should these be `entities` or `metrics` (instead of summary) as the resources were defined above?

Reply:
Entities and metrics. We shouldn't be using the word summary to be consistent. Will change now.

Reply:
Changed everywhere but one spot now where I didn't want to name clash within the module vs. the imports. Everywhere else is changed/fixed now.